src/components/kennels/TrailLocationMap.tsx

```
@@ -0,0 +1,149 @@
+"use client";
+
+import { useMemo, useEffect, useRef } from "react";
+import { APIProvider, Map, useMap, useMapsLibrary } from "@vis.gl/react-google-maps";
```

Biome (2.4.6)

[error] Do not shadow the global "Map" property.

(lint/suspicious/noShadowRestrictedNames)